v1/summary.js

```diff
@@ -15,6 +15,9 @@ module.exports = function(options, templates) {
                     thumb.source = thumb.source.replace(/^http:/, 'https:');
                 }
                 return thumb;
+            },
+            getTimestamp: function(items) {
```

Comment:
I wonder if it would be better to expose a more generalised function like:

```js
getRevisions: function(revItems) {
  if (Array.isArray(revItems) && revItems.length) {
    return revItems;
  }
  return [{}];
}
```